src/main/java/org/snomed/snowstorm/core/data/services/BranchMergeService.java

```diff
@@ -487,6 +487,7 @@ private void fixDuplicateComponentsOfType(String branch, Commit commit, BranchCr
 					// End duplicate components in extension module
 					extensionVersion.setEnd(commit.getTimepoint());
 					repository.save(extensionVersion);
+					BranchMetadataHelper.setRebaseDuplicatesRemoved(commit, extensionVersion.getClass(), Set.of(extensionVersion.getId()));
```

Comment:
This will store the duplicates "in memory" of the commit, which will later be accessed when writing traceability logs.

Reply:
Well spot it.